src/components/input/math-input.js

```diff
@@ -344,6 +330,17 @@ class MathInput extends React.Component {
         });
     };
 
+    _handleScroll = () => {
+        // If animateIntoPosition is false, the user is currently manually positioning
```

Comment:
This makes complete sense. I'm a little bit worried about the increasing complexity around things we're doing manually in this project. That doesn't have anything to do with this change, just on my mind.

Reply:
Yeah, I totally get that. I feel like all the work I'm doing is overriding defaults and creating custom behavior. At least in this change I removed an override as well, so the net complexity for this change is closer to 0 than most of them.

For this specific line, I wouldn't mind renaming the "animateIntoPosition" variable because it took me a while to understand what it even meant.